<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FITNESS APP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
border: 10px solid #FB98AD;background-color: #0C0C0C;
color: #FB98AD;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="create_btn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>450</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1D1D1D;border: 3px solid #FB98AD;
font: 8pt &quot;PRIMETIME&quot;;
</string>
    </property>
    <property name="text">
     <string>CREATE TRAINING</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1D1D1D;border: 3px solid #FB98AD;
font: 8pt &quot;PRIMETIME&quot;;</string>
    </property>
    <property name="text">
     <string>BROWSE TRAININGS</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>77</y>
      <width>97</width>
      <height>97</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FB98AD;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/user.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="profile_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>431</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1D1D1D;font: 20pt &quot;PRIMETIME&quot;;
margin: auto;
border: 5px solid #FB98AD;</string>
    </property>
    <property name="text">
     <string>HI, BLINK!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_goal">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1D1D1D;font: 16pt &quot;PRIMETIME&quot;;
border: 5px solid #FB98AD;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
font: 16pt &quot;PRIMETIME&quot;;</string>
    </property>
    <property name="text">
     <string>YOUR GOAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1D1D1D;border: 3px solid #FB98AD;
font: 8pt &quot;PRIMETIME&quot;;</string>
    </property>
    <property name="text">
     <string>REFRESH</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
